src/main/java/com/everest/display/Display.java

```diff
@@ -0,0 +1,47 @@
+package com.everest.display;
+
+import com.everest.parkinglot.ParkingLot;
+
+import java.util.ArrayList;
+
+public class Display {
+    public ArrayList<ArrayList<Integer>> display(ParkingLot parkingLot, int startIndex, int endIndex){
```

Comment:
Keep the method names proper. It should prints some info as per its name 'display' but it's returning some value